components/eventing-controller/api/v1alpha2/subscription_webhook.go

```diff
@@ -155,6 +170,13 @@ func (s *Subscription) validateSubscriptionSink() *field.Error {
 	return nil
 }
 
+func (s *Subscription) ifKeyExistsInConfig(key string) bool {
+	if _, ok := s.Spec.Config[key]; ok {
+		return true
+	}
+	return false
```

Comment:
No need for the if-statement, the following code is enough:

```golang
func (s *Subscription) ifKeyExistsInConfig(key string) bool {
    _, ok := s.Spec.Config[key]
    return ok
}
```

Reply:
that's right! will change

Reply:
done